siliconcompiler/tools/vpr/vpr.py

```diff
@@ -86,27 +89,37 @@ def runtime_options(chip, tool='vpr'):
         chip.error(f"Specified task {task} doesn't map to a VPR operation", fatal=True)
 
     if 'sdc' in chip.getkeys('input', 'constraint'):
-        sdc_file = f"--sdc_file {chip.get('input', 'constraint', 'sdc', step=step, index=index)}"
-        options.append(sdc_file)
+        sdc_file = find_single_file(chip, 'input', 'constraint', 'sdc',
+                                    file_not_found_msg="SDC file not found")
```

Comment:
why this helper function? find_files will error out if the files are missing and you should ensure the input/constraint/sdc is in the require list if its set so the preprocess will check if it's there.

Reply:
The main purpose of the helper is to hide/reuse the ugliness of checking that exactly one file has been passed to these arguments instead of a whole list of files, since VPR only allows one file to be passed to each of these arguments.